__tests__/app.test.js

```diff
@@ -0,0 +1,63 @@
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  categoryData,
+  commentData,
+  reviewData,
+  userData,
+} = require("../db/data/test-data/index");
+const request = require("supertest");
+const { app, server } = require("../app.js");
+
+beforeEach(() => {
+  return seed({ categoryData, commentData, reviewData, userData });
+});
+
+afterAll(() => {
+  server.close();
```

Comment:
We shouldn't need to interact with our listen while testing as the listen is the 'dev environment' not the 'test environment'